src/MovieLogger.jsx

```diff
@@ -16,7 +19,21 @@ function MovieLogger() {
       .then((data) => setMovies(data))
       .catch((error) => console.error("Error fetching movies:", error));
   };
-
+  function logMovieForUser(movieId) {
+    const rating = 1;
```

Comment:
This is currently hard coded to 1. Is the plan to make it user input eventually?

Reply:
yes, The initial movies that Users log will be hard coded and then from then on they are calculated